src/screens/SocialProfile.js

```diff
@@ -0,0 +1,357 @@
+import React from 'react';
+import { Dimensions, StyleSheet, ScrollView, Platform, Image, ActivityIndicator } from 'react-native';
+import { Font, LinearGradient } from 'expo';
+// galio components
+import { Button, Block, Icon, Text, NavBar, Card } from 'galio-framework';
+import theme from '../theme';
+
+const BASE_SIZE = theme.SIZES.BASE;
+const GRADIENT_GREY = ['#f3f3f3', '#ffffff'];
+const GRADIENT_CRIMSON = ['#f96332', '#fe2472'];
+const COLOR_WHITE = theme.COLORS.WHITE;
+const COLOR_GREY = theme.COLORS.MUTED; 
+const FONT_REGULAR = "SFUIText-Regular";
+const FONT_BOLD = "SFUIText-Bold";
+const { width } = Dimensions.get('screen');
+
```

Comment:
@samcyn you can add a **COLOR_DARK_GREY = '#444444'** and replace the usage from bellow

Reply:
Sure thanks for that.